sql/core/src/test/resources/sql-tests/inputs/datetime-parsing.sql

```diff
@@ -1,5 +1,33 @@
 --- TESTS FOR DATETIME PARSING FUNCTIONS ---
 
+-- parsing with pattern 'y'.
+-- the range of valid year is [1, 294247],
```

Comment:
this is confusing. Spark does support negative year, right?

Reply:
This is wrong, I will fix this, the range is [-290307, 294247]